climakitae/threshold_tools.py

```diff
@@ -25,8 +16,21 @@
 
 
 def get_ams(da, extremes_type="max"):
-    """
-    Returns a data array of annual maximums.
+    """Function that converts data into annual maximums
+
+    Takes input array and resamples annually and takes the maximum value.
```

Comment:
resamples annually *by taking* the maximum value

Reply:
Fixed. Thx!